mymysql/templates/_helpers.tpl

```diff
@@ -0,0 +1,18 @@
+{{/* Generate basic labels */}}
+{{- define "mychart.labels" }}
+  labels:
+    generator: helm
+    name: {{ .Release.Name | printf "%s-%s" .Chart.Name }}
+    chartname: {{ .Chart.Name }}
+    releasename: {{- .Release.Name -}}
+{{- end }}
+
+{{- define "myhelm1.name" -}}
+{{- .Release.Name -}}
+{{- end -}}
+
+{{- define "mychart.matchLabels" }}
```

Comment:
It's better to use the same template in both places metadata.labels and selectors, in order to have in the same state.